toolchain/mfc/count.py

```diff
@@ -12,7 +12,13 @@ def handle_dir(dirpath: str) -> typing.Tuple[typing.List[typing.Tuple[str, int]]
 
     for filepath in glob.glob(os.path.join(dirpath, '*.*f*')):
         with open(filepath) as f:
-            count = sum(1 if not l.isspace() else 0 for l in f.read().split('\n'))
+            count = 0
+            for l in f.read().split('\n'):
```

Comment:
It might be easier to read if this was rewritten as:
```python
for l in f.read().split('\n'):
    # Skip withespace
    if l.isspace() or len(l) == 0:
        continue
    # Skip comments but not !$acc ones!
    if l.lstrip().startswith("!") and not l.lstrip().startswith("!$acc"):
        continue
    count += 1
```

Reply:
Agree

Reply:
resolved via [f5032c7](https://github.com/MFlowCode/MFC/pull/281/commits/f5032c75cde50fa2aa99850c4e163618aa798ae1)